app/src/app/handleLocations.js

```diff
@@ -34,7 +34,13 @@ export function openWindow(url) {
     return
   }
 
-  secondaryWindow = new BrowserWindow()
+  secondaryWindow = new BrowserWindow({
+    webPreferences: {
+      nodeIntegration: url.startsWith('file:'),
```

Comment:
This is way better than before. It still leaves a theoretical attack vector open. Ideally we can turn it off and expose custom functions. That said this might take a week or two to get right.